src/utils/scores.test.ts

```diff
@@ -120,25 +120,45 @@ describe('score tests', () => {
     ORDER BY num_comments DESC
     ;`;
 
-    const discussionCommentersQuery = `
+    const githubCommentersQuery = `
     SELECT
-      discussions.username as username,
-      COUNT(discussions.username) as num_comments,
+      issues.username as username,
+      COUNT(issues.username) as num_comments,
     FROM
-      \`open_source.github_events\` AS discussions
+      \`open_source.github_events\` AS issues
     WHERE
-      discussions.type = 'discussion_comment'
+      (issues.type = 'discussion_comment' OR issues.type = 'issue_comment')
       AND timestamp_diff(
         CURRENT_TIMESTAMP(),
-        discussions.created_at,
+        issues.created_at,
+        day
+      ) <= 7
+      AND issues.user_type != 'external'
+      AND issues.user_type != 'bot'
+    GROUP BY issues.username
+    ORDER BY num_comments DESC
+    ;`;
+
+    const issuesQuery = `
```

Comment:
nit: `issueCommentsQuery`. Took me a while that we're grabbing the comments, not the issues themselves.

Same for the tests below.

Reply:
done